axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/bankreconciliation/load/BankReconciliationLoadService.java

```diff
@@ -75,32 +75,36 @@ public void loadBankStatementLines(
    * @return the filter.
    */
   protected String getBankStatementLinesFilter(
-      boolean includeOtherBankStatements, boolean includeBankStatement) {
+      boolean includeOtherBankStatements,
+      boolean includeBankStatement,
+      boolean isLineTypeMovement) {
```

Comment:
Please create a separate method that will call this one instead of adding a boolean, this will improve code readability